src/components/OrganizationDetailsTable.tsx

```diff
@@ -2,7 +2,9 @@ import React from "react";
 import { Table, Spin, Tag, Alert } from "antd";
 import { SortOrder } from "antd/lib/table/interface";
 import { useQuery } from "react-apollo";
-import { fracToPercent, formatDate } from "utils/displayUtils";
+import { BarType } from "./PercentBar";
```

Comment:
The project has avoided relative imports so far. No strong feelings here, but for consistency I'd suggest changing this to `"components/PercentBar"`

Reply:
Handled in 59aa6e5.